go-controller/pkg/cni/cni.go

```diff
@@ -345,6 +349,10 @@ func HandlePodRequest(
 // instance of the pod in the apiserver, see checkCancelSandbox for more info.
 // If kube api is not available from the CNI, pass nil to skip this check.
 func getCNIResult(pr *PodRequest, getter PodInfoGetter, podInterfaceInfo *PodInterfaceInfo) (*current.Result, error) {
+	if err := checkBridgeMapping(pr.toplogy, pr.netName); err != nil {
```

Comment:
You already have the topology defined in the CNI conf. Did you try using it and it didn't work ? 

```suggestion
	if err := checkBridgeMapping(pr.CNIConf.Topology, pr.netName); err != nil {
```